parachain-template/node/src/command.rs

```diff
@@ -287,6 +287,15 @@ pub fn run() -> Result<()> {
 			let collator_options = cli.run.collator_options();
 
 			runner.run_node_until_exit(|config| async move {
+				let hwbench = if !cli.no_hardware_benchmarks {
+					config.database.path().map(|database_path| {
+						let _ = std::fs::create_dir_all(&database_path);
```

Comment:
If this fails then the node won't start anyway since the node won't be able to create the database, and the disk benchmarks itself will print out an error that the directory doesn't exists too. So you'll have two errors printed out already; there's no point in printing out a third one.